lucene/analysis/common/src/test/org/apache/lucene/analysis/hunspell/TestDutchIJ.java

```diff
@@ -27,6 +27,5 @@ public static void beforeClass() throws Exception {
   public void testStemming() {
     assertStemsTo("ijs", "ijs");
     assertStemsTo("IJs", "ijs");
-    assertStemsTo("Ijs");
```

Comment:
Hunspell/C++ stems that a bit differently from Lucene, but in a controversial way, so I removed the check instead of changing the expectation

Reply:
Ok.